helmfile/values/dex.yaml.gotmpl

```diff
@@ -1,4 +1,4 @@
-imageTag: "v2.26.0"
+imageTag: "v2.28.1"
```

Comment:
This doesn't seem to exist anymore, it's moved to `image.tag`
Although I think we can remove it and just use the default, which is the same as this.